### PR TITLE
Introduce static analysis

### PR DESCRIPTION
**Pull request by szepeviktor**
Was #9035, this goes to master, see comments in the other PR.

### CODE REVIEW

File: src/Control/IPUtils.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-namespace SilverStripe\Control\Util;
+namespace SilverStripe\Control;
```
Comment (robbieaverill) [inline src/Control/IPUtils.php]: Would you mind adding a old-to-new mapping entry in `.upgrade.yml` for this change?

Comment (szepeviktor) [reply src/Control/IPUtils.php]: Done.

**Comment by szepeviktor** [comment]
🐴

**Comment by sminnee** [comment]
We should get this in the `4` branch

**Pull request merged**